created python script for dataset description json template

anushkab opened the pull request:
resolved issue https://github.com/INCF/bids-starter-kit/issues/19

choldgraf commented:
hey @anushkab and @dorahermes - I think these are certainly nice issues to get started on for building tools. That said, we want to make sure not to duplicate effort too much, so keep an eye on this PR: https://github.com/mne-tools/mne-bids/pull/28 which implements some of the functions that you're describing in this issue. In particular, see https://github.com/mne-tools/mne-bids/pull/28/files#diff-57fc2a5d75c57bb143f494b0637ca740R145

choldgraf commented:
(once that PR is merged, we'll still need to add some code here explaining how to use it, which it'd be great if you could help with @anushkab !)

emdupre commented:
Thanks, @choldgraf ! Yes, I think this is something to keep in mind generally for differentiating these two resources.

dorahermes commented:
Thank you @anushkab ! 

I'll be coordinating the Matlab code and will leave the Python experts (@choldgraf, @emdupre and @KirstieJane ) in charge of approving/merging Python code.

In terms of overlapping code - the starters kit is only meant to have the very basic code, just to create one specific file (this pull request looks good to me).  There will be several packages (Fieldtrip/MNE) that are creating code to create a more complete structure.

dorahermes commented on templates/dataset_description.json:
```diff
@@ -1,23 +1,23 @@
 {
```
This file does not need to be changed

anushkab replied:
The file had some indentation change as per the python code. I have now uploaded file dataset_description.json as it was before.

anushkab commented:
Thank you @emdupre for suggesting the changes. 

I have updated the files with some code comments. The code now follows PEP8 conventions.
 Please let me know if further change is required.

Regards
Anushka

anushkab commented:
Thank you @emdupre for suggesting the changes.
I have updated the files and made sure code follows PEP8 conventions using http://pep8online.com/checkresult
I have also added the code comments as suggested
Please let me know if any change has to be made.

Regards
Anushka

dorahermes commented:
lgtm

Remi-Gau commented:
@all-contributors please add @emdupre  for review

Remi-Gau commented:
@all-contributors please add @anushkab for code content ideas
